docs/api/models.rst

```diff
@@ -175,14 +175,14 @@ NAIP
    :file: weights/naip.csv
 
 
-Sentinel-1
-----------
+Synthetic Aperture Radar (SAR)
+------------------------------
```

Comment:
I disagree with this change. There are many different kinds of SAR (C-band vs. L-band) just like there are many different kinds of optical imagery (Sentinel-2, Landsat, etc.). Just because two models have the same number of bands doesn't mean they will be compatible with any SAR imagery.

Reply:
That's okay with me but you're not proposing a solution here. Do you prefer that I just make a separate group for "Umbra Space" in this case? Because this isn't Sentinel-1.